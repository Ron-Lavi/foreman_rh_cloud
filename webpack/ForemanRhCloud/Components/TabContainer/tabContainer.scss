.tab-content {
  padding: 20px;
  min-height: 0;

  .row {
    margin-bottom: 20px;
  }
}
