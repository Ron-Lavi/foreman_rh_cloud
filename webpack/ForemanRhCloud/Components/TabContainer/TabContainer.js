import React from 'react';
import PropTypes from 'prop-types';
import { Grid } from 'patternfly-react';
import './tabContainer.scss';

const TabContainer = ({ children, className }) => (
  <div className={className}>
    <Grid>
      <Grid.Col>{children}</Grid.Col>
    </Grid>
  </div>
);

TabContainer.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string,
};

TabContainer.defaultProps = {
  children: null,
  className: '',
};

export default TabContainer;
