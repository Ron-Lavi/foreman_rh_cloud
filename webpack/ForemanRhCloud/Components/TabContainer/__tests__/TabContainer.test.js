import { testComponentSnapshotsWithFixtures } from '@theforeman/test';

import TabContainer from '../TabContainer';

const fixtures = {
  'render without Props': {},
  /** fixtures, props for the component */
};

describe('TabContainer', () => {
  describe('rendering', () =>
    testComponentSnapshotsWithFixtures(TabContainer, fixtures));
});
