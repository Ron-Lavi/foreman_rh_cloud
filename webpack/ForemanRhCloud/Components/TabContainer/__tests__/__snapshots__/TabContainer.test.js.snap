// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabContainer rendering render without Props 1`] = `
<div
  className=""
>
  <Grid
    bsClass="container"
    componentClass="div"
    fluid={false}
  >
    <Col
      bsClass="col"
      componentClass="div"
    />
  </Grid>
</div>
`;
